skycultures/western_sternenkarten/info.ini

```diff
@@ -0,0 +1,6 @@
+[info]
+name = Western (Sternenkarten)
+author = Edgar Scholz
+descriptionSource = https://sternenkarten.com/die-sternbilder/
+boundaries = iau

```

Comment:
Please add license info here, for example: `license = CC BY-ND 4.0 International`